cmake/templates/pkgConfig.cmake.in

```diff
@@ -123,6 +123,22 @@ foreach(library ${libraries})
     list(APPEND @PROJECT_NAME@_LIBRARIES ${library})
   elseif(${library} MATCHES "^-l")
     list(APPEND @PROJECT_NAME@_LIBRARIES ${library})
+  elseif(${library} MATCHES "^-")
+    # This is a linker flag/option (like -pthread)
+    # There's no standard variable for these, so create an interface library to hold it
+    if(NOT @PROJECT_NAME@_NUM_DUMMY_TARGETS)
+        set(@PROJECT_NAME@_NUM_DUMMY_TARGETS 0)
+    endif()
+    MATH(EXPR VAR "${@PROJECT_NAME@_NUM_DUMMY_TARGETS}+1")
```

Comment:
This line is a no op since the result is assigned to the variable `VAR` which isn't used anywhere. I assume the intention was to increment `@PROJECT_NAME@_NUM_DUMMY_TARGETS`?

Nitpick: the CMake function name should be lower case to match the style guide.

Reply:
lowercase and actually used output in 64295ca7f6a3079f4a15ee3ff5985f038639ec99